libnavui-androidauto/src/main/java/com/mapbox/androidauto/car/feedback/core/CarFeedbackPollProvider.kt

```diff
@@ -0,0 +1,178 @@
+package com.mapbox.androidauto.car.feedback.core
+
+import androidx.annotation.DrawableRes
+import androidx.car.app.CarContext
+import androidx.car.app.model.CarIcon
+import androidx.core.graphics.drawable.IconCompat
+import com.mapbox.androidauto.R
+import com.mapbox.androidauto.car.feedback.ui.CarFeedbackOption
+import com.mapbox.androidauto.car.feedback.ui.CarFeedbackPoll
+import com.mapbox.navigation.core.telemetry.events.FeedbackEvent
+import com.mapbox.search.analytics.FeedbackEvent.FeedbackReason
+
+open class CarFeedbackPollProvider {
```

Comment:
```suggestion
/**
 * Each screen contains contextual feedback options. This class contains default
 * items, override the functions to provide custom feedback options.
 */
open class CarFeedbackPollProvider {
```

Will you add some javadocs here? These are public interfaces that will can be referenced through the documentation

https://docs.mapbox.com/android/navigation/api/2.6.0/libnavui-androidauto/com.mapbox.androidauto.car.feedback.core/-car-feedback-item-provider/